PyOpenWorm/dataObject.py

```diff
@@ -240,15 +250,15 @@ def __repr__(self):
 
 
 def DatatypeProperty(*args, **kwargs):
-    return APThunk('DatatypeProperty', args, kwargs)
+    return APThunk(DatatypeProperty.__name__, args, kwargs)
```

Comment:
I'd like these to change back to literals. I'm not seeing any advantage to indirection here.